yearn/hardhat.config.ts

```diff
@@ -0,0 +1,72 @@
+import { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
```

Comment:
To use `localNetworksConfig` property we need to add an import here:
```js
import "@keep-network/hardhat-local-networks-config"
```

And in `tsconfig.json` add `"node_modules/@keep-network/hardhat-local-networks-config/src/type-extensions.d.ts"` to `files` list.

Reply:
https://github.com/keep-network/tbtc-v2/pull/14/commits/436541e53f3fe038491632ce2639c9b6f44db132